<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="sys_info">
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <attribute name="title">
      <string>system infomation</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>host name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hostName">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>NAME</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>time of system startup:</string>
      </property>
     </widget>
     <widget class="QLabel" name="startUpTime">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TIME</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>system uptime:</string>
      </property>
     </widget>
     <widget class="QLabel" name="uptime">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TIME</string>
      </property>
     </widget>
     <widget class="QLabel" name="frequency">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>300</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>FREQUENCY</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpu">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>250</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>MODEL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>system version:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU model:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>300</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU frequency:</string>
      </property>
     </widget>
     <widget class="QLabel" name="version">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>VERSION</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pro_info">
     <attribute name="title">
      <string>process</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>551</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="new_pro">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>70</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_run">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>run </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="targrt_pro">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>160</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>190</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>serch</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="kill_pro">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>250</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_kill">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>280</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>kill</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="cur_info">
     <attribute name="title">
      <string>performance</string>
     </attribute>
     <widget class="QCustomPlot" name="cpu_graphic" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>691</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Memory</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="mem_graphic" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>691</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Swap</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="swap_graphic" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>691</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="poweroff">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>480</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>power off</string>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="cpu_used">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="mem_used">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
